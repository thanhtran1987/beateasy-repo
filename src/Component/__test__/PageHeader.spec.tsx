import * as React from 'react';
import { shallow } from 'enzyme';

import TestComponent from '../PageHeader';

describe('PageHeader', () => {
  it('should render correctly in default state', () => {
    const component = shallow(<TestComponent />);
    expect(component).toMatchSnapshot();
  });
});
