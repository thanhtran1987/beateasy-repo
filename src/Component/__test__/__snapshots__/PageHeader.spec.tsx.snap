// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageHeader should render correctly in default state 1`] = `
<styled.div>
  <styled.div>
    <Styled(Component) />
  </styled.div>
  <styled.div>
    <ul>
      <li>
        <span>
          <Component />
        </span>
        <h1>
          Racing
        </h1>
      </li>
    </ul>
  </styled.div>
</styled.div>
`;
